mysql-test/t/percona_bug_ps290.test

```diff
@@ -0,0 +1,9 @@
+# PS-290: Unsafe query warning improvements caused a crash
```

Comment:
The test should go to binlog suite, and SET binlog_format replaced with have_binlog_format_statement.inc